proxy/src/http/source.rs

```diff
@@ -345,7 +349,7 @@ mod handler {
     ) -> Result<impl Reply, Rejection> {
         let peer = peer.lock().await;
         let urn = project_urn.parse().map_err(Error::from)?;
-        let branches = coco::with_browser(&peer, &urn, |browser| coco::branches(browser))?;
+        let branches = coco::with_browser(&peer, &urn, |browser| coco::branches(browser, None))?;
```

Comment:
We might not want to list remote branches here. This is probably not an issue right now because we have no remote.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/636/commits/c276f923533cb491f634f557c38c757d4e669985